scene/2d/gpu_particles_2d.h

```diff
@@ -67,6 +68,16 @@ class GPUParticles2D : public Node2D {
 
 	void _update_particle_emission_transform();
 
+#ifdef TOOLS_ENABLED
+	Vector2 vis_rect_hvframes;
+	CanvasItemMaterial *material_in_use;
+
+	virtual void set_material(const Ref<Material> &p_material) override;
+	virtual void set_use_parent_material(bool p_use_parent_material) override;
```

Comment:
Those two should be `public:` like in `CanvasItem`.

Reply:
changed!